test/ray_julia_jll/put_get.jl

```diff
@@ -11,13 +11,17 @@ using .ray_julia_jll: ObjectID
     @testset "roundtrip vector" begin
         data = UInt16[1:3;]
         buffer = LocalMemoryBuffer(Ptr{Nothing}(pointer(data)), sizeof(data), true)
-        oid = ray_julia_jll.put(RayObject(buffer), StdVector{ObjectID}())
-        @test ray_julia_jll.contains(oid)
+        oid = CxxPtr(ray_jll.ObjectID())
+        status = ray_julia_jll.put(RayObject(buffer), StdVector{ObjectID}(), oid)
+        @test Symbol(status) == :OK
+        @test ray_julia_jll.contains(oid[])
 
         # TODO: Currently uses size/length from `data`
         # https://github.com/beacon-biosignals/Ray.jl/issues/55
-        ray_obj = ray_julia_jll.get(oid, -1)
-        buffer = GetData(ray_obj[])
+        ray_obj = CxxPtr(SharedPtr{ray_jll.RayObject}())
+        status = ray_julia_jll.get(oid[], -1, ray_obj)
+        @test Symbol(status) == :OK
```

Comment:
Testing it this way ensures that a `Status` type is returned or at least comparable
```suggestion
        @test status == Status(:OK)
```
Will need to import `Status`

Reply:
ah yeah I was trying to figure out the inverse of that and was using `StatusCode` 👍

Reply:
I think the difference is that the Enum is a `StatusCode` while the return of the function is a `Status`

Reply:
alt:
```julia
julia> ray_julia_jll.ok(status)
true

julia> ray_julia_jll.code(status) == ray_julia_jll.StatusCode(:OK)
true
```